Description: Fix crash with GNOME applications by adding an extra check
 while reading value for color space.
Author: Frederic Peters <fpeters@debian.org>
Bug-Debian: http://bugs.debian.org/398426
Last-Update: 2011-04-11
--- a/libexif/exif-entry.c
+++ b/libexif/exif-entry.c
@@ -1317,7 +1317,7 @@
 		/* Find the value */
 		for (j = 0; list2[i].elem[j].values[0] &&
 			    (list2[i].elem[j].index < v_short); j++);
-		if (list2[i].elem[j].index != v_short) {
+		if (list2[i].elem[j].index != v_short || v_short == 65535) {
 			snprintf (val, maxlen, _("Internal error (unknown "
 				  "value %i)"), v_short);
 			break;
